terraso_backend/apps/core/models/taxonomy_terms.py

```diff
@@ -41,6 +41,7 @@ def __str__(self):
         return "{}.{}".format(self.type, self.slug)
 
     class Meta(SlugModel.Meta):
+        _ignore_unique_fields = ["slug"]
```

Comment:
Question: I am not sure if I am able to understand all the changes, maybe this question will help. What is the difference between having here: `_unique_fields = []` or `_ignore_unique_fields = ["slug"]`?

Reply:
Yes, that's a good question. 

The metaclass takes the unique fields from each parent class to gather the list of constraints it should create. 

```python
        all_unique_fields = set(
            attrs.get("_unique_fields", [])
            + [field for base in bases for field in getattr(base, "_unique_fields", [])]
        )
```

I did this so that we could concatenate the lists throughout the class category, rather than the child class overwriting the parent class. However that means that there is no obvious way for the child class to skip over a unique field listed in the parent class. An easy solution is just to list explicitly if you would like to ignore a unique field declared in the parent class.

Reply:
Thanks David